### PR TITLE
chore(cvo): report progressing=true if the version has updated

### PR DESCRIPTION
**Pull request by ecordell**

### CODE REVIEW

File: pkg/lib/operatorstatus/status.go [hunk 1]
```diff
@@ -131,17 +140,24 @@ func MonitorClusterStatus(name string, syncCh chan error, stopCh <-chan struct{}
...
 						Name:    "operator",
-						Version: version,
+						Version: targetOperatorVersion,
 					},
 					{
 						Name:    "operator-lifecycle-manager",
 						Version: olmversion.OLMVersion,
 					},
 				}
+				if targetOperatorVersion != previousOperatorVersion {
```
Comment (njhale) [inline pkg/lib/operatorstatus/status.go]: Just to check my understanding - on cluster init, `previousOperatorVersion` _should_ be "0.0.1-snapshot" here, right?

Comment (ecordell) [reply pkg/lib/operatorstatus/status.go]: it should be, yes

**Pull request merged**